phpstan.baseline.neon

```diff
@@ -54,3 +54,18 @@ parameters:
 		 message: "#^Match arm is unreachable because previous comparison is always true.$#"
 		 count: 1
 		 path: src/Http/Pool.php
+
+		-
+		 message: "#^Method Saloon\\\\Http\\\\Faking\\\\MockClient\\:\\:getRequestClass\\(\\) should return class-string|null but returns class-string<Saloon\\\\Http\\\\Request>|Saloon\\\\Http\\\\Request$#"
+		 count: 1
+		 path: src/Http/Faking/MockClient.php
+
+		-
+		 message: "#^Call to an undefined method ReflectionType\\:\\:getName\\(\\)\\.$#"
+		 count: 1
+		 path: src/Http/Faking/MockClient.php
+
+		-
+		 message: "#^Parameter \\#1 \\$function of class ReflectionFunction constructor expects Closure|string, callable\\(\\)\\: mixed given\\.$#"
+		 count: 1
+		 path: src/Http/Faking/MockClient.php
```

Comment:
```suggestion
```

Reply:
Without them, phpstan doesn't pass: 

![CleanShot 2024-08-05 at 02 14 13](https://github.com/user-attachments/assets/5e1e4b04-3f27-4150-8f48-2d56865a9bc7)

I believe they are false-positive.

Reply:
They aren't false positives, but the question is if they need addressing or if it should be considered an invalid usage by the user.